### ISSUE TITLE
Kubernetes Dashboard errors

### ISSUE THREAD

**Issue by kvpt** [Azure/AKS#494]
Hi,

I created a Kubernetes cluster few days ago using AKS, it worked well, but today I can't use the dashboard anymore.
When accessing dashboard, this spam 404 errors from the api, see below.

![dashboard](https://user-images.githubusercontent.com/1446221/42129903-5268877e-7cd3-11e8-83c6-09b0af51fd2d.gif)

On the other hand, kubectl commands work fine.

**Comment by sauryadas** [comment]
if you are still seeing this issue , i recommend filing a support ticket

**Comment by kvpt** [comment]
Still the same.
I am currently using credits included with my MPN subscription for testing the managed kubernetes solution, and technical support are not included in this offer.
I am in a dead-end.

**Comment by sauryadas** [comment]
Did you enable RBAC in the cluster? If yes, follow steps in the link below to fix it
https://pascalnaber.wordpress.com/2018/06/17/access-dashboard-on-aks-with-rbac-enabled/

**Comment by kvpt** [comment]
Yes I have enabled RBAC to use with RabbitMq chart provided by Bitnami.
I followed the documentation on their web site :  https://docs.bitnami.com/azure/get-started-aks/#optional-configure-helm-to-use-charts-with-rbac

I tried the solution described in your link with no success.
As it is for testing I think I will recreate a new cluster. 

Have you an official documentation to enable RBAC correctly in Azure ?
